net_handler/udp_conn.c

```diff
@@ -94,6 +94,7 @@ static void* send_device_data(void* args) {
                     }
                     param__init(param);
                     param->name = device_info->params[j].name;
+                    param->readonly = device_info->params[j].write == 0;
```

Comment:
Can we add a TODO comment or something after this to make sure we know to use `stdbool.h` once we get around to using that header? And to make it more explicit, I usually prefer writing it as
```
device_info->params[j].write ? 0 : 1;
```

Reply:
I made it check for `false` now. I also don't like using the ternary operator for this.

Reply:
I think it's more explicit; also if you don't specifically set it to 1, the true value could be any nonzero value, which can cause problems up the line when packing the protobuf / interpreting the boolean value.

Reply:
The C language will only return 1 or 0 for an == operation, so you don't need to worry about that.